src/DimpleB/Assignment4program14.java

```diff
@@ -0,0 +1,64 @@
+/* WAP to print output as below
+ Please enter string:
+Input:Hi Good Morning All
+Output:llA GNINROM doog HI
+ */
+
+package DimpleB;
+
+import java.util.Scanner;
+
+public class Assignment4program14 {
+
+	void reversalstring(String str)
```

Comment:
HI @DimpleBAug19 
For some value it throw an ArrayIndexOutOfBoundException.
like..dimple technocredits java selenium